include/klee/BoundedMergeHandler.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
Added the standard header with the KLEE license and a doxygen comment at the top explaining the functionality (taken from this PR).